docker/install.sh

```diff
@@ -14,6 +14,6 @@
 #  License for the specific language governing permissions and limitations under
 #  the License.
 
-source /opt/ros/kinetic/setup.bash
+source /opt/ros/noetic/setup.bash
```

Comment:
Update copyright here and in other files

Reply:
Updated for this file. The other files are originally BSD license from torc robotics which was the sub which wrote these. So I'm not really sure if they should be changed

Reply:
That makes sense, we can leave those as is then.